Modules/_operator.c

```diff
@@ -993,12 +1005,26 @@ itemgetter_call(itemgetterobject *ig, PyObject *args, PyObject *kw)
     PyObject *obj, *result;
     Py_ssize_t i, nitems=ig->nitems;
 
-    if (!_PyArg_NoKeywords("itemgetter", kw))
-        return NULL;
-    if (!PyArg_UnpackTuple(args, "itemgetter", 1, 1, &obj))
-        return NULL;
-    if (nitems == 1)
+    assert(PyTuple_CheckExact(args));
+    if (kw == NULL && PyTuple_GET_SIZE(args) == 1) {
```

Comment:
`_PyArg_NoKeywords` is a macro, no need to "optimize" it manually.

Reply:
I prefer this as-is.  There is some tiny duplication which gets be compiled away, but to my eyes the `kw==NULL` check makes it clearer what the common path is and that no external calls are made.  Also, I don't trust that `_PyArg_NoKeywords` will remain a macro.